opm/simulators/utils/ParallelRestart.hpp

```diff
@@ -76,7 +76,7 @@ namespace Action {
 
 class Aqudims;
 class BCConfig;
-class BCConfig::BCFace;
+struct BCConfig::BCFace;
```

Comment:
this is useless. 
- you are prototyping an existing structure (ie already included)
- you can never prototype a member of another class like this in any case.

Reply:
(I already remove this in https://github.com/OPM/opm-simulators/pull/2305 so my preferred fix is for you not to touch it, so i avoid the rebase)

Reply:
OK - done.